### PR TITLE
ISPN-16672 Avoid blocking calls in lsof in server report

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-16672

### CODE REVIEW

File: server/runtime/src/main/server/bin/report-osx.sh [hunk 1]
```diff
@@ -32,13 +32,13 @@ uname -a > "$TMPDIR/uname"
 df -h > "$TMPDIR/df"
 lsof -itcp -P -n | grep "LISTEN"  > "$TMPDIR/ss-tcp"
 lsof -iudp -P -n > "$TMPDIR/ss-udp"
-lsof / > "$TMPDIR/lsof" 2> /dev/null
+lsof -b / > "$TMPDIR/lsof" 2> /dev/null
```
Comment (jabolina) [inline server/runtime/src/main/server/bin/report-osx.sh:35]: There's a couple lsof calls above, should the have the option, too?

**Review by jabolina — COMMENTED** [review]
Blocks only on mac?

**Comment by tristantarrant** [comment]
> Blocks only on mac?

It blocks on both, but I added the fix only to one of the scripts (even though I don't have a mac

File: server/runtime/src/main/server/bin/report-osx.sh (hunk 1 above)
Comment (tristantarrant) [reply server/runtime/src/main/server/bin/report-osx.sh:35]: I'll ask a mac owner to test that

**Comment by tristantarrant** [comment]
@jabolina I've added `-b` to all the lsof invocations

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Thanks, @tristantarrant!

**Pull request merged**